File: src/main/java/aquarius0715/commandcountplugin/DateFormant.java
package aquarius0715.commandcountplugin;


import java.text.SimpleDateFormat;
import java.util.Date;


public class DateFormant {

    CommandCountPlugin plugin;

    public DateFormant(CommandCountPlugin plugin) {
        this.plugin = plugin;
    }

    public void FormStartTime() {
        Date date = new Date();
        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy'-'MM'-'dd' 'HH':'mm':'ss");
        plugin.StartDate = simpleDateFormat.format(date);
    }
}

File: src/main/java/aquarius0715/commandcountplugin/DateFormant.kt
package aquarius0715.commandcountplugin

import java.text.SimpleDateFormat
import java.util.*

class DateFormant(var plugin: CommandCountPlugin) {
    fun StartTime() {
        val date = Date()
        val simpleDateFormat = SimpleDateFormat("yyyy'-'MM'-'dd' 'HH':'mm':'ss")
        plugin.StartDate = simpleDateFormat.format(date)
    }

}